demos/sev-demo/README.md

```diff
@@ -0,0 +1,136 @@
+
+We can step through the keysets file.
+```
+{
+   "KEYSET-1":{
+```
+Currently we are hardcoding our keyset in the Kata configuration file (see above) to match `KEYSET-1` specified here. 
+```
+      "allowed_digests":[
+         "a11cc9f32b3e64a0846b5eadf773e5e4ee7b4187d984b7a72a3b4fc99e34374c",
+         "dcf332c8abb68cc534d7f1e523ffe9e072308e0d92dc98e3884b313caaea379b"
+      ],
+```
+Firmware digests that are allowed for this keyset. You can calculate a firmware digest using the `calculate_hash.py` tool. Be sure to specify the correct, kernel, initrd, firmware, and kernel parameters. These will be included in the hash (this is an extension of the original SEV behavior). You should have the path to these components in your kata configuration file. Do not use the kernel parameters field from the configuration file, though, because it only contains additional parameters rather than he full command line. The easiest way to get the full kernel parameters is to start a container with Kata and then check the QEMU arguments via `journalctl -xe -u containerd` or `dmesg | grep qemu`. Note that viewing the kernel commandline from inside the guest via `cat /proc/cmdline` might not accurately reflect the firmware's view of the kernel commandline, which is what the measurement is based on.  
+```
+      "allowed_keys":[
+         "key_id1"
+      ],
+```
+This should correspond to the key in `keys.json`.
+```
+      "allowed-policies":[
+         0
+      ],
+      "min-api-major":0,
+      "min-api-minor":0,
+      "allowed-build-ids":[
+         13
+      ]
+```
+These parameters should be familiar if you have launched any SEV guests before. Note that you can set multiple allowed policies or allowed build IDs for each keyset.
+```
+   }
+}
+```
+
+Once you have the KBS/GOP configuration files setup, simply run the KBS.
+
+## Preparing an image 
+
+Finally, it's time to prepare an image. You can use the `offline_fs_kbs` sample KBS as a keyprovider. It is compatible with `offline_sev_kbc`. See the [README](https://github.com/confidential-containers/attestation-agent/blob/main/sample_kbs/src/enc_mods/offline_fs_kbs/README.md) for guidance. You should end up using a skopeo command that looks something like this. 
+```
+OCICRYPT_KEYPROVIDER_CONFIG=ocicrypt.conf skopeo copy docker://docker.io/<username>/<some-container>:unencrypted docker://docker.io/<username>/<some-container>:encrypted --encryption-key provider:attestation-agent:keys.json:key_id1
+```
+Make sure the sample KBS is running per the linked instructions. You may need to run `skopeo login`. `key_id1` in `keys.json` is exactly the key that needs to be added to `keys.json` in the KBS directory.
```

Comment:
Some random thoughts:

- There are a lot of (potentially error-prone) steps the user has to follow.
  Could any of this be automated by a script or two?

- Is it worth adding a "Common Problems" section?

- Do you have a feeling for how long it would take to complete this demo?

- It might be useful to have a "Conclusion" section that summarises what the user should have been able to do (summary of expected results).

Reply:
Our end goal is to support SEV via Kata Deploy and the operator. This guide is partly to help facilitate that development. The process is highly error-prone and very time consuming so it really shouldn't be undertake by too many end users. 

I'll add an overview section that highlights all of the changes. Maybe at the beginning.